R/analyse.R

```diff
@@ -2,18 +2,113 @@
 
 
 
-#' Title
+#' Analyse Multiple Imputed Datasets
 #'
 #' @description
-#' TODO
+#' This function takes multiple imputed datasets (as generated by
+#' the [impute()] function) and runs an analysis function on
+#' each of them.
 #'
-#' @param imputations TODO
-#' @param fun TODO
-#' @param delta TODO
-#' @param ... TODO
+#' @details
+#' This function works by performing the following steps
 #'
+#' 1. Extract a dataset from the `imputations` object
+#' 2. Apply any delta adjustments as specified by the `delta` argument
+#' 3. Run the analysis function `fun` on the dataset
+#' 4. Repeat steps 1-3 across all of the datasets inside the `imputations`
+#' object
+#' 5. Collect and return all of the analysis results.
+#'
+#' The analysis function `fun` must take a data.frame as its first
+#' argument. All other options to [analyse()] are passed onto `fun`
+#' via `...`.
+#' `fun` must return a named list with each element being itself a
+#' list containing a single
+#' numeric element called `est` (or additionally `se` and `df` if using
+#' you had originally specified [method_bayes()] or [method_approxbayes()])
```

Comment:
Sorry here maybe the wording is not correct (maybe without "using" sounds better?)

Reply:
Yup sorry thats a typo, I have pushed a fix